src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -13,8 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import React, { useRef, useState, useEffect } from 'react';
+import React, { ReactNode, useEffect, useState, useRef } from 'react';
```

Comment:
Type import here as well for `ReactNode`

Reply:
That's done